### PR TITLE
fix - 325 Converted the dependent microsevices api host format supported for openshift

### PR DESCRIPTION
**Pull request by riginoommen**
# Fixes - 325

### 

# Explain the fix
Changed the supported microservices environemt variables as same as the openshift variables

## Does this PR introduce a breaking change

No

## Screenshots

NA
<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/api-gateway-service/.env.example [hunk 1]
```diff
@@ -21,4 +21,5 @@ COOKIE_PARSER=
 
 ## Dependent Microservices
 ## User Group
-# USER_GROUP="https://localhost:8080/graphql"
+# USER_SERVICE_SERVICE_HOST="https://localhost:8080/graphql"
+# HOME_SERVICE_SERVICE_HOST="https://localhost:8081/graphql"
```
Comment (deshmukhmayur) [inline packages/api-gateway-service/.env.example:25]: Remove the protocol and `/graphql`. The openshift env variables will just have the hostname.

```suggestion
# USER_SERVICE_SERVICE_HOST="localhost:8080"
# HOME_SERVICE_SERVICE_HOST="localhost:8081"
```

Comment (deshmukhmayur) [reply packages/api-gateway-service/.env.example:25]: Also, add the examples for other microservices, such as Feedback and Notifications.

File: packages/api-gateway-service/src/helpers.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ const fetch = require( 'node-fetch' );
 
 /* This consists of the list of microservices which needs to be part of the gateway */
 export const serviceList = [
-  { name: 'User Group', url: process.env.USER_GROUP }
+  { name: 'User Service', url: `http://${ process.env.USER_SERVICE_SERVICE_HOST }/graphql` },
```
Comment (deshmukhmayur) [inline packages/api-gateway-service/src/helpers.ts:5]: Add other microservices to the list.

Comment (riginoommen) [reply packages/api-gateway-service/src/helpers.ts:5]: Added

File: packages/api-gateway-service/.env.example (hunk 1 above)
Comment (riginoommen) [reply packages/api-gateway-service/.env.example]: Added other examples

**Comment by riginoommen** [comment]
> Will bring up the https config change during standup.
> You can remove it from the the `service.ts` according to that discussion.
> Or else we might have to change the `helper.ts` to use https protocol.

We are changing the helper.ts

**Pull request merged**